src/python/turicreate/_deps/README.md

```diff
@@ -0,0 +1,186 @@
+# Lazy import layout
+
+---
+
+## Rationale
+
+The import time (wall time) is more than 2 seconds, which is extremely slower than state-of-the-art tools' import speed. At the time of this writing, the import wall time of sklearn is around 108 ms.
```

Comment:
Put in a detail with this statistic: The import time of version 5.7 was 2 seconds on a (...) ...

Reply:
got it.